pkg/apis/serving/v1/revision_lifecycle.go

```diff
@@ -171,6 +171,18 @@ func (rs *RevisionStatus) PropagateDeploymentStatus(original *appsv1.DeploymentS
 func (rs *RevisionStatus) PropagateAutoscalerStatus(ps *autoscalingv1alpha1.PodAutoscalerStatus) {
 	// Reflect the PA status in our own.
 	cond := ps.GetCondition(autoscalingv1alpha1.PodAutoscalerConditionReady)
+	if ps.ActualScale != nil {
+		rs.ActualReplicas = *ps.ActualScale
```

Comment:
Could we use the [GetActualScale helper](https://github.com/knative/serving/blob/1640d2755a7c61bdb65414ef552bfb511470ac70/pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go#L279) here and avoid the nil check?